tests/installation/ssh_host_key_settings/verify_ssh_key_not_imported.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: verify that SSH host key dit not got imported.
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    x11_start_program('xterm -geometry 150x45+5+5', target_match => 'xterm');
+    become_root;
+    script_run 'cd /etc/ssh';
+    enter_cmd "cat /etc/ssh/ssh_host_key | tee /dev/$serialdev";
+    wait_serial "SSHHOSTKEYFILE", 5, 1 || die "/etc/ssh/ssh_host_key content does match";
+    enter_cmd "cat /etc/ssh/ssh_host_key.pub | tee /dev/$serialdev";
+    wait_serial "SSHHOSTPUBKEYFILE", 5, 1 || die "/etc/ssh/ssh_host_key.pub content does match";
+    script_run "md5sum * | tee /dev/$serialdev";
```

Comment:
I know, I was also puzzled first. You need to see this in the context of the test suite. There is a test at the beginning that sets the files for the SSH host keys. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/c8223c263e0b18c8bafd7fe7345c69d3ae9dbf2e/tests/x11/ssh_key_check.pm#L16
Since we can't verify the existence of real SSH keys the test module just writes plain text in those files and this checks for that plain text.